### PR TITLE
add butler repo symlinks

### PR DESCRIPTION
**Pull request by jhoblitt**

### CODE REVIEW

File: hieradata/site/cp/role/nfsserver.yaml [hunk 1]
```diff
@@ -41,6 +45,29 @@ nfs::nfs_exports_global:
       139.229.167.0/24(rw,nohide,insecure,no_subtree_check,async,root_squash)
       139.229.170.0/24(rw,nohide,insecure,no_subtree_check,async,root_squash)
 
+files:
+  /data/project/shared/auxTel:
+    ensure: "directory"
+    mode: "2775"
+    owner: 0
+    gropu: 768  # jovyan, in the LSP context
```
Comment (adrienthebo) [inline hieradata/site/cp/role/nfsserver.yaml]:
```suggestion
    group: 768  # jovyan, in the LSP context
```

File: hieradata/site/cp/role/nfsserver.yaml [hunk 2]
```diff
@@ -41,6 +45,29 @@ nfs::nfs_exports_global:
...
+  /project/shared/auxTel/_parent:
+    ensure: "link"
+    target: "/readonly/repo"
+  /project/shared/auxTel/raw:
+    ensure: "link"
+    target: "/readonly/repo/raw"
```
Comment (ktlim) [inline hieradata/site/cp/role/nfsserver.yaml]: This should be removed.

File: hieradata/site/cp/role/nfsserver.yaml [hunk 3]
```diff
@@ -41,6 +45,29 @@ nfs::nfs_exports_global:
...
+  /project/shared/auxTel/rerun:
+    ensure: "directory"
+    mode: "2775"
+    owner: 0
+    gropu: 768  # jovyan, in the LSP context
```
Comment (adrienthebo) [inline hieradata/site/cp/role/nfsserver.yaml]:
```suggestion
    group: 768  # jovyan, in the LSP context
```

File: hieradata/site/cp/role/nfsserver.yaml [hunk 4]
```diff
@@ -41,6 +45,29 @@ nfs::nfs_exports_global:
...
+  /project/shared/auxTel/CALIB:
+    ensure: "directory"
+    mode: "2775"
+    owner: 0
+    gropu: 768  # jovyan, in the LSP context
```
Comment (adrienthebo) [inline hieradata/site/cp/role/nfsserver.yaml]:
```suggestion
    group: 768  # jovyan, in the LSP context
```

File: hieradata/site/cp/role/nfsserver.yaml [hunk 5]
```diff
@@ -4,6 +4,10 @@ classes:
   - "profile::core::nfsserver"
   - "profile::core::nfsclient"
 
+lookup_options:
```
Comment (adrienthebo) [inline hieradata/site/cp/role/nfsserver.yaml]: Nonblocker: If possible we should push this lookup to `manifests/site.pp`

Comment (jhoblitt) [reply hieradata/site/cp/role/nfsserver.yaml]: I agree, that definitely makes sense.

File: hieradata/site/cp/role/nfsserver.yaml (hunk 1 above)
Comment (jhoblitt) [reply hieradata/site/cp/role/nfsserver.yaml]: eeww, gawd...

**Comment by jhoblitt** [comment]
Looks like I missed the `/data` prefix on several of the paths too. Sheesh.

**Pull request merged**